src/app/components/layout/side-menu/side-menu.component.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <mat-list>
+    <a [routerLink]="''">
+      <mat-list-item class="link"><span>{{'SIDE_MENU.MAIN_PAGE' | translate}}</span></mat-list-item>
+    </a>
+    <a [routerLink]="'personal-data'">
```

Comment:
Can we store links in some other file for example 'routing-links'? If we would like to change for example 'requests-list' to 'application-list' or something, we have to change it here and in routing, otherwise menu will stop working. Let me know what you think ;)

Reply:
Done! Routes are now stored only in the `app-routing.module.ts` (like they were before) and they can be imported as an array to any component if needed using `@angular/router`.